app/route_rangers_api/utils/heatmap_data_prep.py

```diff
@@ -0,0 +1,218 @@
+# Purpose: Query database to format and prepare demographic data
+# for heatmaps
+# Author: Jimena Salinas
+# Date: Day 17th, 2024
+
+import os
+import sys
+import django
+from dotenv import load_dotenv
+import pandas as pd
+from shapely import wkt
+import geopandas as gpd
+from geopandas import GeoDataFrame
+import json
+from shapely.geometry import MultiPolygon
+
+
+def setup_django() -> None:
```

Comment:
If you are running things via `python -m manage ....` which is how this will be ingested in production, you just need to import everything directly from `route_rangers_api` instead of `app.route_rangers_api` so all of your imports should be able to be at the top and line 18-27 can be deleted